python/src/sensor.py

```diff
@@ -0,0 +1,139 @@
+import time
+from datetime import datetime
+
+import uuid
+import random
+import dognames
+import names
+import argparse
+from db import config
+from db.client import ScyllaClient
+
+
+def generate_random_pet_name():
+    return dognames.male() if random.randint(1, 2)==1 else dognames.female()
+
+
+def generate_random_measurement_value(sensor_type):
+    if sensor_type == "T":
+        return 101 + random.uniform(0, 10) - 4;
+    elif sensor_type == "L":
+      return 35 + random.uniform(0, 5) - 2
+    return random.uniform(0, 20)
+
+
+def generate_random_measurement(sensor):
+    return {
+        "sensor_id": sensor["sensor_id"],
+        "ts": datetime.now(),
+        "value": generate_random_measurement_value(sensor["type"])
+    }
+  
+  
+def create_owner():
+    return {
+        "owner_id": uuid.uuid4(),
+        "address": "home",
+        "name": names.get_full_name() # generate random person's name
+    }
+
+
+def create_pet(owner):
+    return {
+        "owner_id": owner["owner_id"],
+        "pet_id": uuid.uuid4(),
+        "address": "home",
+        "age": random.randint(1, 20),
+        "name": generate_random_pet_name(),
+        "weight": random.uniform(5.0, 15.0)
+    }
+
+
+def create_sensors(pet):
+    return [
+        {
+            "pet_id": pet["pet_id"],
+            "sensor_id": uuid.uuid4(),
+            "type": "T"
+        },
+        {
+            "pet_id": pet["pet_id"],
+            "sensor_id": uuid.uuid4(),
+            "type": "L"
+        }
+    ]
+
+def insert_pet_static_data(client: ScyllaClient, pet, owner, sensors):
```

Comment:
you have type annotation for `client`, it would be nice that other parameters would have them too